script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -824,7 +824,7 @@ If the lines are blood vessels,
 -- Page 486, Offset 17780.
 -- それは本来ありえない運動よ。脳は過負荷を起こして、志貴は間違いなく使い物にならなくなる」
 But in reality, that is an impossible task.
- If you overwork your brain like that, you will definitely cease to be useful."
+ If you overwork your brain like that, you will without a doubt, cease to be useful."// Just wanted to check if Arcueid is more saying that Shiki would become a vegetable or if she's more saying that he wouldn't be useful for vampire hunting anymore if he overworks himself. I do see that they clarify it in a later line so might be fine to leave it as is
```

Comment:
Yeah it's intentionally vague here to allow the subsequent dialog. 

Not sure I love this use of without a doubt, it doesn't really feel right with the pause structure it's used in here.